extensions/mssql/src/objectManagement/ui/principalDialogBase.ts

```diff
@@ -184,6 +185,8 @@ export abstract class PrincipalDialogBase<ObjectInfoType extends SecurityPrincip
 			});
 			const data = this.getSecurableTableData();
 			await this.setTableData(this.securableTable, data);
+		} else {
+			await vscode.window.showInformationMessage(localizedConstants.SchemaIsEmptyInfoMessage);
```

Comment:
Any particular reason why you're using the global notification message here rather than the dialog notification message (i.e. this.dialogObject.message)?